docs/spec_draft.md

```diff
@@ -1791,6 +1792,55 @@ and produces a `result` tensor.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.power
+
+### Semantics
```

Comment:
Have you considered what happens with negative `lhs` and negative `rhs`?

Reply:
The behaviour is standard mathematical behaviour and illegal expressions result in NANs. Let me know if you feel we need to write something in this regard.

Reply:
That's fine for floating-point numbers (and for complex numbers too, for which we consistently say "corner cases TBD" in the spec). What about integers though? NaN won't cut it for integers.

Reply:
negative power for integers greater than 1 results in 0
negative integers in `lhs` work as expected because fractional powers are not permitted due to C1
0 raised to negative power causes runtime exception.

Reply:
divide by zero resulting in exception is expected behaviour and everything else also seems expected behaviour.

Reply:
"negative power for integers greater than 1 results in 0". This was not expected behavior for me. Does this mean that this computation is performed in floating point and then rounded towards negative infinity or something?

"divide by zero resulting in exception is expected behaviour". This is inconsistent with https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md#errors which says "error conditions are [...] possible, e.g. through integer overflows, out-of-bounds accesses, etc. Unless explicitly called out, all these errors result in implementation-defined behavior".

Overall, it looks like we need to explicitly document the behavior of raising integers to negative powers with three distinct cases: 1) less than one, 2) one, 3) greater than one.